src/App/Fossa/FossaAPIV1.hs

```diff
@@ -24,9 +24,11 @@ module App.Fossa.FossaAPIV1 (
   getSignedURL,
   archiveUpload,
   archiveBuildUpload,
+  assertUserDefinedBinaries,
 ) where
 
 import App.Fossa.Container (ContainerScan (..))
+import App.Fossa.IAT.Types
```

Comment:
[nit] We are moving towards, [not having implicit blanket imports](https://github.com/fossas/spectrometer/blob/style-guide/STYLE-GUIDE.md#dont-use-implicit-blanket-imports). 

```suggestion
import App.Fossa.IAT.Types (Fingerprint (..), UserDefinedAssertionMeta(..))
```